Dockerfile

```diff
@@ -17,10 +17,10 @@ RUN apk add --no-cache \
       build-base \
       libffi-dev
 # Install Radicale
-ADD https://github.com/Kozea/Radicale/archive/${VERSION}.tar.gz radicale.tar.gz
-RUN tar xzf radicale.tar.gz && \
+RUN wget --quiet https://github.com/Kozea/Radicale/archive/${VERSION}.tar.gz && \
```

Comment:
Good catch, I wouldn't have saw that coming for a million bucks. I made the change.